gruntfile.js

```diff
@@ -1,30 +1,27 @@
-let _ = require('underscore');
-
 module.exports = function (grunt) {
 
-  grunt.loadNpmTasks('grunt-contrib');
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-concat');
-
+  // Project
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
-    concat: {
-      dist: {
-        src: ['public/js/jquery.min.js', 'public/bootstrap.min.js', 'public/js/app.js'],
-        dest: 'public/js/<%= pkg.name %>.add.js'
-      }
-    },
-    uglify: {
-      options: {
-        banner: '/*! <%= pkg.name %> <%= grunt.template.today("dd-mm-yyyy") %> */\n'
-      },
-      dist: {
-        files: {
-          'public/js/<%= pkg.name %>.min.js': ['<%= concat.dist.dest %>']
-        }
-      }
-    }
+    sass: {
```

Comment:
Please fix the indent here. I think it got bad because of copy paste
We use 2 space indent here.